mlrun/config.py

```diff
@@ -650,17 +654,27 @@ def read_env(env=None, prefix=env_prefix):
             cfg = cfg.setdefault(name, {})
         cfg[path[0]] = value
 
+    env_db_path = env.get("MLRUN_DBPATH", "")
+    is_remote_mlrun = env_db_path.startswith("https://mlrun-api.")
```

Comment:
This heuristic is not good enough, we may have the db path with this prefix in a marketplace instance of the kit (which doesn't have v3io)